ts/bootstrap.tsx

```diff
@@ -24,6 +25,7 @@ declare var window: any;
 window['_promise'] = Promise;
 window['_classlist'] = ClassList;
 window['_url'] = URL;
+window['_fetch'] = fetch;
```

Comment:
Yeah, the compiler will ignore the import unless it's being referenced in the file.  I think I can just mark the polyfill as exported, but this way also gives us a window through the browser to see if the polyfill worked.